microsoft/vscode
# add option to not show link hovers for terminal

Adds an option to disable showing link hovers in the terminal.



Testing
1. Open a terminal and having a link inside of it. (`echo "http://localhost:3000"`).
2. Hover the link and see that a hover displays.
3. Modify the setting `terminal.integrated.showLinkHover` to be false.
4. Hover the link again and see that a hover does not display.

This PR fixes https://github.com/microsoft/vscode/issues/113913#issuecomment-755363179, but not the entire issue of hovers not disappearing.

Fixes #125282
